### PR TITLE
Made the webapp more responsive

### PR DESCRIPTION
**Pull request by YvonneAriri**
Description:

- Made the app more responsive to make it easy for the users to use your website regardless of what device they're using. 

Milestone:

- Implemented the app to be more responsive. This milestone ensures that the app can be used on various screen sizes and resolutions.

Test Plan:

Eveprep9

### CODE REVIEW

File: event-app-ui/src/components/Profile/Profile.jsx [hunk 1]
```diff
@@ -57,6 +59,20 @@ export default function Profile() {
...
+    const filterFutureEvent = eventData.filter((event) => {
+      const eventStartTime = new Date(event.starttime);
+      return eventStartTime > currentDate;
+    });
+    const filterPastEvent = eventData.filter((event) => {
+      const eventEndTime = new Date(event.endtime);
+      return eventEndTime < currentDate;
+    });
+    setFutureEvent(filterFutureEvent);
+    setPastEvent(filterPastEvent);
```
Comment (sophia129) [inline event-app-ui/src/components/Profile/Profile.jsx:73]: Let's say an event is ongoing, where the start time is less than now and end time is greater than now. Would event show up here?

File: event-app-ui/src/components/Profile/Profile.jsx [hunk 2]
```diff
@@ -111,23 +127,92 @@ export default function Profile() {
...
-                    );
-                  }
-                  return null;
-                })}
-              </div>
+              <>
+                <h2 className="future-past">Future Events</h2>
+                <div className="event-view">
+                  {futureEvent.map((value, key) => {
+                    if (value.username === username) {
```
Comment (sophia129) [inline event-app-ui/src/components/Profile/Profile.jsx:134]: 1. Are you going to differentiate how past and future events look in the UI? Right now these 2 sections are exactly the same.
2. From the previous diff, it's not ideal to filter events on the front end. I expect this logic to be fixed in a subsequent PR.
3. However, L131-171 and L172-211 are the same so you can generate both using a .map() function too.

No action needed for now, just FYI

**Review by sophia129 — CHANGES_REQUESTED** [review]
Wow, UI looks so much better!
Don't forget to mention all your code changes in the PR description. You also add logic to differentiate past and future events in this PR.

File: event-app-ui/src/components/Profile/Profile.jsx (hunk 1 above)
Comment (YvonneAriri) [reply event-app-ui/src/components/Profile/Profile.jsx:73]: I'll create a section for ongoing events in the next pull request

**Pull request merged**